ddtrace/contrib/logging/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""
+The patching of standard library logging to inject tracing information
```

Comment:
Yeah, this feels weird here, I think we should move this into `advanced_usage.rst`

Reply:
Otherwise we'll be pulling it in as an integration... which it... sort of is ?

Reply:
We might want to be a little more explicit with our description here letting them know what tracing information we submit, and how to add to their format string.

We also want an example of using `DD_LOGS_INTEGRATION=true` as well